### PR TITLE
[vnet] fix leftovers: remove 'INTERFACE' entry from redis and add VLAN_MEMBER back in Cleanup block

### PR DESCRIPTION
**Pull request by ayurkiv-nvda**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
fix leftovers that remained after test execution

#### How did you do it?
I  removed remained vnet related config and added Ethernet4 interface back to VLAN1000
#### How did you verify/test it?
run` py.test vxlan/test_vnet_vxlan.py --inventory "../ansible/inventory, ../ansible/veos" --host-pattern arc-switch1041-t0 --module-path ../ansible/library/ --testbed arc-switch1041-t0 --testbed_file ../ansible/testbed.csv --skip_sanity --disable_loganalyzer -v  --log-cli-level info`   <--- this test may add leftovers (before this fix)
run ` py.test iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_config --inventory "../ansible/inventory, ../ansible/veos, ../ansible/lab" --host-pattern arc-switch1041-t0 --module-path ../ansible/library/ --testbed arc-switch1041-t0 --testbed_file ../ansible/testbed.csv --allow_recover  --assert plain --log-cli-level info --show-capture=no -ra --showlocals -v `
Test will fail with message "test fails to add interface to vlan 100"
After fix test should pass

### CODE REVIEW

File: tests/vxlan/vnet_utils.py [hunk 1]
```diff
@@ -146,6 +146,9 @@ def cleanup_dut_vnets(duthost, mg_facts, vnet_config):
     for vnet in vnet_config['vnet_id_list']:
         duthost.shell("redis-cli -n 4 del \"VNET|{}\"".format(vnet))
 
+    for intf in vnet_config['intf_list']:
+        duthost.shell("redis-cli -n 4 del \"INTERFACE|{}|{}\"".format(intf['ifname'], intf['ip']))
```
Comment (volodymyrsamotiy) [inline tests/vxlan/vnet_utils.py:150]: This command just removes IP address from the RIF.
But still need to remove RIF itself (same as it is done for ```VLAN_INTERFACE``` few lines above).
So something like shown below should be added after current line:
```
duthost.shell("redis-cli -n 4 del \"INTERFACE|{}\"".format(intf['ifname']))
```

File: tests/vxlan/test_vnet_vxlan.py [hunk 2]
```diff
@@ -95,23 +95,23 @@ def vxlan_status(setup, request, duthosts, rand_one_dut_hostname, vnet_test_para
...
-        attached_vlan = mg_facts["minigraph_vlan_interfaces"][0]['attachto']
-        member_to_remove = mg_facts["minigraph_vlans"][attached_vlan]['members'][0]
-        duthost.shell("redis-cli -n 4 del \"VLAN_MEMBER|{}|{}\"".format(attached_vlan, member_to_remove))
+        duthost.shell("redis-cli -n 4 del \"VLAN_MEMBER|{}|{}\"".format(attached_vlan, vlan_member))
 
         apply_dut_config_files(duthost, vnet_test_params)
 
         vxlan_enabled = True
     elif request.param == "Cleanup" and vnet_test_params[CLEANUP_KEY]:
+        duthost.shell("redis-cli -n 4 hset \"VLAN_MEMBER|{}|{}\" \"tagging_mode\" \"untagged\" ".format(attached_vlan, vlan_member))
```
Comment (volodymyrsamotiy) [inline tests/vxlan/test_vnet_vxlan.py:113]: Can we avoid hard-coded value for "tagging_mode"?
Maybe get from config before removing, store it and then use it to restore default?

**Comment by volodymyrsamotiy** [comment]
Please add ```[vnet]``` prefix to the commit message and PR title

File: tests/vxlan/vnet_utils.py (hunk 1 above)
Comment (ayurkiv-nvda) [reply tests/vxlan/vnet_utils.py:150]: Done

File: tests/vxlan/test_vnet_vxlan.py (hunk 2 above)
Comment (ayurkiv-nvda) [reply tests/vxlan/test_vnet_vxlan.py]: Done

**Review by volodymyrsamotiy — APPROVED** [review]

**Pull request closed without merging**